app/services/foreman_seeder.rb

```diff
@@ -19,11 +19,7 @@ def foreman_seeds
 
   def plugin_seeds
     Foreman::Plugin.registered_plugins.collect do |name, plugin|
-      engine = (name.to_s.tr('-', '_').camelize + '::Engine').constantize
-      Dir.glob(engine.root + 'db/seeds.d/*.rb')
-    rescue NameError => e
-      Foreman::Logging.exception("Failed to register plugin #{name}", e)
-      nil
+      Dir.glob(plugin.engine.root + 'db/seeds.d/*.rb') if plugin.engine
```

Comment:
Would it make sense to add a helper in the plugin itself to return this glob? Like `plugin.seeds`

Reply:
unsure about this one :thinking: we don't need it anywhere else, and there is the same path for foreman few lines above this one. I don't have preference tho, I can abstract it to the plugin if it makes more sense :)

Reply:
I'm fine with this the way it is